From 76e4a976a51ab6f7ec066326242210752c002777 Mon Sep 17 00:00:00 2001
From: Drew Fustini <drew@beagleboard.org>
Date: Sun, 30 May 2021 23:35:04 -0700
Subject: [PATCH 52/56] riscv: dts: starfive-jh7100: remove reg-names from gpio
 node

Remove the uneeded reg-names property from the gpio controller node.

Signed-off-by: Drew Fustini <drew@beagleboard.org>
---
 arch/riscv/boot/dts/starfive/jh7100.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/riscv/boot/dts/starfive/jh7100.dtsi b/arch/riscv/boot/dts/starfive/jh7100.dtsi
index debbe0e4e4eb..89832c281e8d 100644
--- a/arch/riscv/boot/dts/starfive/jh7100.dtsi
+++ b/arch/riscv/boot/dts/starfive/jh7100.dtsi
@@ -347,13 +347,12 @@ usb3: usb@104c0000 {
 
 		gpio: gpio@11910000 {
 			compatible = "starfive,jh7100-gpio";
-			interrupts = <32>;
 			reg = <0x0 0x11910000 0x0 0x10000>;
-			reg-names = "control";
-			interrupt-controller;
-			#interrupt-cells = <2>;
 			gpio-controller;
 			#gpio-cells = <2>;
+			interrupt-controller;
+			#interrupt-cells = <2>;
+			interrupts = <32>;
 		};
 
 		i2c0: i2c@118b0000 {
-- 
2.30.2

